content/docs/guides/components/mxnet.md

```diff
@@ -0,0 +1,152 @@
+# MXNet Training
```

Comment:
@suleisl2000 You need a header in the markup to ensure hugo can generate corresponding title entries and add it to the navigation etc. Please see the header at some other component guides in this directory for example.

Reply:
@abhi-g Comment is addressed. Please help to check it again.